src/sbus/codegen/sbus_CodeGen.py

```diff
@@ -22,8 +22,9 @@
 import argparse
 from collections import OrderedDict
 from sbus_Introspection import Introspectable
-from sbus_Template import *
-from sbus_Generator import *
+from sbus_Template import TemplateFile
+from sbus_Generator import Generator
+from sbus_DataType import DataType
```

Comment:
I already tried this some months ago and the compilation failed. I think you are facing the same issue in you [builds](https://github.com/SSSD/sssd/runs/6707953608?check_suite_focus=true).

Reply:
I see now. I will remove this part of change from the codegen.